service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java

```diff
@@ -97,4 +98,34 @@ public void test_getMultipleValuesAsString_expectTrimmedValues() {
     //then
     Assert.assertEquals("test / me / please", output);
   }
+
+  @Test
+  public void test_getActivityCARGO_AIRPLANEAsString_expectReadableValue() {
+    //given
+    Activity activity = Activity.CARGO_AIRPLANE;
+
+    String activityName = BeaconsStringUtils.enumAsString(activity);
+    //then
+    Assert.assertEquals("Cargo Airplane", activityName);
+  }
+
+  @Test
+  public void test_getActivityHOT_AIR_BALLOONAsString_expectReadableValue() {
+    //given
+    Activity activity = Activity.HOT_AIR_BALLOON;
+
+    String activityName = BeaconsStringUtils.enumAsString(activity);
+    //then
```

Comment:
Consider removing comments. What do these add that's not already communicated by your code?

Reply:
Fair point, I think it's a habit to enforce separation. 

If more lines were to come, it makes it clear of the cause -> effect of the test